### ISSUE TITLE
Base classes with extension methods should be purple in the package containing the method

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#497]
![image](https://user-images.githubusercontent.com/623951/62333214-70e9f700-b477-11e9-9925-883cc78a205f.png)

The code in ``RowanPackageService>>update`` shows both classes having extension methods as shown below. Therefore, wethinks both should be purple. 

Note - GsFile is not packaged. 

```
	| loadedPackage |
	loadedPackage := Rowan image loadedPackageNamed: 'st_launcher-GemStone-Kernel' ifAbsent:[^KeyValueDictionary new].
	^loadedPackage loadedClassExtensions aKeyValueDictionary( 'FileDoesNotExistException'->aRwGsLoadedSymbolDictClassExtension FileDoesNotExistException, 'GsFile'->aRwGsLoadedSymbolDictClassExtension GsFile)
```

**Comment by LisaAlmarode** [comment]
Rowan adds a certain number of methods to the base, unpackaged classes.
When the package that contains the extension method is selected, the base classes for which there are extension methods should be present in the classes list, and purple.

The classes are there (I think it wasn't once, but I can't find the example now), but they are definitely not purple.

Another example in 3.0.74 is Cypress | Cypress-Environemental-Tools | Behavior | persistentSuperclassForEnv:

**Comment by ericwinger** [comment]
Should have a fix for this in 3.0.75.
